lact-daemon/src/server/gpu_controller/mod.rs

```diff
@@ -639,7 +639,12 @@ impl GpuController {
                 }
             }
 
-            hw_mon.set_power_cap(cap)?;
+            // TODO: Due to possible driver bug, RX 7900 XTX really doesn't like when we set the same value again.
```

Comment:
I don't think it makes sense leaving this as a TODO - even if the issue with reapplying the value gets fixed on the driver side, there's no harm in keeping this check to avoiding unnecessarily writing the power cap.

Same in the `else` case

Reply:
I removed TODO note, but general message is still there. I think in the future it makes sense to apply all changes incrementally.